### PR TITLE
Reimplement "Add user setting to choose between words based random alias and uuid alias"

### PR DESCRIPTION
**Pull request by doanguyen**
this PR follows #5 + reformat

### CODE REVIEW

File: app/dashboard/views/index.py [hunk 1]
```diff
@@ -99,9 +102,9 @@ def index():
 
     client_users = (
         ClientUser.filter_by(user_id=current_user.id)
-        .options(joinedload(ClientUser.client))
-        .options(joinedload(ClientUser.gen_email))
-        .all()
+            .options(joinedload(ClientUser.client))
```
Comment (nguyenkims) [inline app/dashboard/views/index.py]: the indentation here isn't the one used by black I think.

**Comment by nguyenkims** [comment]
Overall the PR is almost perfect! Just a small fix on the model (`server_default` column) and running black before pushing and we can merge it.

**Comment by doanguyen** [comment]
I did not know black is so handful :D, the file is then updated

File: app/models.py [hunk 2]
```diff
@@ -83,13 +84,23 @@ class PlanEnum(enum.Enum):
...
+
+
 class User(db.Model, ModelMixin, UserMixin):
     __tablename__ = "users"
     email = db.Column(db.String(128), unique=True, nullable=False)
     salt = db.Column(db.String(128), nullable=False)
     password = db.Column(db.String(128), nullable=False)
     name = db.Column(db.String(128), nullable=False)
     is_admin = db.Column(db.Boolean, nullable=False, default=False)
+    alias_generator = db.Column(db.Integer, nullable=False, default=AliasGeneratorEnum.word.value)
```
[the remark this answers is not in the document]
Comment (nguyenkims) [reply app/models.py]: in case you haven't seen it ...

[the remark this answers is not in the document]
Comment (doanguyen) [reply app/models.py]: Oops, I've made the update in the last commit

**Pull request merged**